Scripts/sims4communitylib/utils/common_omega_utils.py

```diff
@@ -0,0 +1,32 @@
+from distributor.ops import Op
+from protocolbuffers.Consts_pb2 import MGR_UNMANAGED, MSG_OBJECTS_VIEW_UPDATE
+from protocolbuffers.DistributorOps_pb2 import Operation
+from protocolbuffers.Distributor_pb2 import ViewUpdate
+from server.client import Client
+from sims4communitylib.utils.misc.common_game_client_utils import CommonGameClientUtils
+
+
+class CommonOmegaUtils:
+    """ Utilities for sending messages to the client via the Omega channel. (Improved S4MP compatibility)
+
+    Messages sent through this channel don't replicate to the rest of the clients in a multiplayer S4MP session.
+    """
+
+    @classmethod
+    def send_view_update_message(cls, op: Op, manager_id=MGR_UNMANAGED, object_id=0, client: Client = None):
+        import omega
```

Comment:
Is "omega" a vanilla script? If not, then this whole function should be surrounded by a "try: except: pass", since omega may not always be available.

Reply:
`omega` is a native Sims 4 module for which I have not been able to find documentation. Some uses of it by the base code can be seen at EA/simulation/server.client.Client.send_message